src/index.html

```diff
@@ -84,13 +84,13 @@ <h3 class="section__h3--title">Activity</h3>
         <section class="activity--container--1 widget--container">
           <article class="activity__article--1 article--widget">
             <h5>TODAY'S ACTIVITY</h5>
-            <img class="exercise-icon" src="./images/exercise.icon.png">
+            <img class="exercise-icon" alt="An icon depicting running man in a green circle" src="./images/exercise.icon.png">
             <div id="daily-activity" class="article__widget--div"></div>
           </article>
-          <img class="step-icon" src="./images/step-icon.png">
+          <img class="step-icon" alt="An icon depicting a foot running" src="./images/step-icon.png">
           <article class="activity__article--2 article--widget">
             <h5>WORLD COMPARISON</h5>
-            <img class="globe-icon" src="./images/globe-icon.png">
+            <img class="globe-icon" alt="An icon depicting the earth as a globe" src="./images/globe-icon.png">
```

Comment:
It's always difficult coming up with the right words for alt tags for me, but these are fairly accurate and descriptive.